aroma/tests/test_integration.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+import os
+import pandas as pd
+import subprocess
+from os.path import join, split, isfile
+from argparse import Namespace
+
+from aroma import main
+
+import pytest
+
+
+def test_integration(skip_integration, nilearn_data):
+    if skip_integration:
+        pytest.skip('Skipping integration test')
+
+    test_path, _ = split(nilearn_data.func[0])
+
+    out_path = join(test_path, 'out')
+
+    confounds = pd.read_csv(nilearn_data.confounds[0], sep='\t')
+
+    mc = confounds[["rot_x", "rot_y", "rot_z", "trans_x", "trans_y", "trans_z"]]
+    mc_path = join(test_path, 'mc.tsv')
+    mc.to_csv(mc_path, sep='\t', index=False, header=None)
+
+    args = Namespace(TR=None, affmat='', denType='nonaggr', dim=0, generate_plots=False, inFeat=None, inFile=nilearn_data.func[0], mask='',
```

Comment:
You need to override the default TR with 2, since the files in the development dataset had their headers overwritten.

Reply:
The code won't run with `TR=2` . It will raise an error in [line 329](https://github.com/eurunuela/ica-aroma-org/blob/test/original/aroma/utils.py#L329) of `utils`.

I'm going to switch off for today. Feel free to finish this PR (I think it's pretty much there). I'll catch up tomorrow first thing in the morning.

Reply:
Some of those `old_div` were the problem. The latest commit fixes that and I made sure the test passes now.